src/NUnitFramework/framework/Internal/Commands/TimeoutCommand.cs

```diff
@@ -121,8 +121,9 @@ private Task<TestResult> RunTestOnSeparateThread(TestExecutionContext context)
         {
             var separateContext = new TestExecutionContext(context)
             {
-                CurrentResult = context.CurrentTest.MakeTestResult()
+                //CurrentResult = context.CurrentTest.MakeTestResult()
```

Comment:
This resets the `OutWriter` as the new test result  will create it's own writer, which when assigned to `separateContext.CurrentResult` will then apply it to the `separateContext`'s outwriter based on https://github.com/nunit/nunit/blob/321db1ae2b67de0049ee9f534518369786d9b557/src/NUnitFramework/framework/Internal/TestExecutionContext.cs#L208-L217

Reply:
@manfred-brands I'd like to pick up fixing this issue, but I'm looking for advice here. Do you recall why we create a new `CurrentResult` in the `TimeoutCommand`?

If we need it then I think the way to fix the issue here is to pass in `context.OutWriter` when creating it to avoid creating new one on initialization. That's a bit involved to do in a non-breaking way so I wanted to double-check first about what the commented out line here does